### PR TITLE
Add and document missing jobtype_include_os_environ in jobtypes.yml

### PR DESCRIPTION
**Pull request by opalmer**
This PR adds a missing configuration option to `jobtypes.yml`.

### CODE REVIEW

**Comment by opalmer** [comment]
@guidow, PTAL

File: pyfarm/jobtypes/etc/jobtypes.yml [hunk 1]
```diff
@@ -70,6 +70,12 @@ jobtype_ignore_id_mapping_errors: False
 # by a job type.
 jobtype_default_environment: {}
 
+# If True then a job type's get_environment() method
+# will also include the operating system's environmnt.
```
Comment (guidow) [inline pyfarm/jobtypes/etc/jobtypes.yml]: Typo (environmnt -> environment)

File: pyfarm/jobtypes/etc/jobtypes.yml [hunk 2]
```diff
@@ -70,6 +70,12 @@ jobtype_ignore_id_mapping_errors: False
...
+# The environment is constructed at the time the agent
+# is launched, modifications to the agent's environemnt
```
Comment (guidow) [inline pyfarm/jobtypes/etc/jobtypes.yml]: Another typo (environemnt -> environment)

**Comment by opalmer** [comment]
Fixed, PTAL.  #353 will need to merge first though before this passes.

**Pull request merged**